docs/web-apps/automated-testing/selenium/selenium4.md

```diff
@@ -148,9 +150,21 @@ SafariOptions browserOptions = new SafariOptions();
 </TabItem>
 </Tabs>
 
-#### Full Example
+List of [W3C WebDriver standard capabilities](https://www.w3.org/TR/webdriver1/#capabilities):
+* `browserName`
+* `browserVersion`
+* `platformName`
+* `acceptInsecureCerts`
+* `pageLoadStrategy`
+* `proxy`
+* `timeouts`
+* `unhandledPromptBehavior`
+
+Any capability not in this list must include a vendor prefix. This rule applies to browser-specific capabilities as well as Sauce Labs-specific capabilities. For example, if you want use the `build` and `name` capabilities in for your Sauce Labs tests, you'll need to wrap them in a `sauce:options` block ([see the examples below](/web-apps/automated-testing/selenium/selenium4/#platform-configuration-examples)).
 
-Below are code examples showing the comparison between the usage of legacy `DesiredCapabilities` and then how how to update it with `browserOptions`. Notably, platform and version are replaced by `platformName` and `browserVersion`, respectively, and Sauce-specific capabilities are placed inside a `sauce:options` block.
+#### Platform Configuration Examples
+
+Below are examples comparing the usage of legacy `DesiredCapabilities` with the new `browserOptions`.
```

Comment:
@titusfortner the example with the `Map<String, Object> sauceOptions = new HashMap<>();` seems overl complicated, could we just create a simpler example without this for Java?